package.json

```diff
@@ -44,16 +44,16 @@
   },
   "devDependencies": {
     "editorconfig-checker": "4.0.2",
-    "eslint": "8.0.1",
+    "eslint": "8.2.0",
     "eslint-plugin-html": "6.2.0",
     "eslint-plugin-import": "2.25.2",
-    "eslint-plugin-unicorn": "37.0.1",
-    "eslint-plugin-vue": "7.19.1",
-    "jest": "27.3.0",
+    "eslint-plugin-unicorn": "38.0.0",
```

Comment:
Should add the [new rules](https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v38.0.0) in `.eslintrc`, I think both can be enabled, e.g. `[2]`.

Reply:
My main point here is get monaco-0.30.0 in as soon as possible. Does this definitely build?

Reply:
`lint-frontend` will fail if the new rules fail, I think it's unlikely that it will.

If you just want to upgrade monaco alone, you could only upgrade it alone, e.g. `npm install monaco-editor@latest monaco-editor-webpack-plugin@monaco-editor-webpack-plugin` on a fresh branch.